index.js

```diff
@@ -5,6 +5,7 @@
 var EventEmitter = require('events').EventEmitter;
 var spawn = require('child_process').spawn;
 var path = require('path');
+var findModuleBin = require('find-module-bin');
```

Comment:
Can you please inline the module, we trying to keep commander.js dependencies-free

Reply:
As a curiosity – What's the reason for wanting no dependencies?

Reply:
find-module-bin uses global-modules, to look up the appropriate os-dependent location of globally installed node modules. this won't be a simple pull-function-up refactor, as the os-specific logic is hidden in here. 

perhaps we can merge this and set the refactoring as tech-debt?

personally, I think the reuse is better for the community. you don't want a massive explosion of dependencies, but having well managed dependencies can make your code more DRY and keeps the potential for the dependent packages to continue to evolve and solidify.

Reply:
@vanesyan

Reply:
@vanesyan I don't think it makes sense to inline all of this. 

This PR has been ready for merge for about a year and six months.

Reply:
Hi @vanesyan @abetomo
My team is also interested in this fix.  

Is there any reason why this can not be included with the dependency?